packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -450,19 +456,17 @@ export default defineComponent({
 
       try {
         const path = join(this.item, folderName)
-        const resource = await (this.$clientService.webdav as WebDAV).createFolder(this.space, {
+        let resource = await (this.$clientService.webdav as WebDAV).createFolder(this.space, {
```

Comment:
Can stay a `const` like it was before, right? You're only setting the `indicators` property further down. The `resource` variable is never reassigned.